lib/portlayer/storage/vsphere/volume.go

```diff
@@ -191,3 +193,22 @@ func (v *VolumeStore) VolumesList(op trace.Operation) ([]*storage.Volume, error)
 
 	return volumes, nil
 }
+
+func (v *VolumeStore) StatPath(op trace.Operation, deviceId string, target string) (stat *compute.FileStat, err error) {
+	diskDsURI := v.volDiskDsURL(deviceId)
+	mountPath, err := v.dm.AttachAndMount(op, diskDsURI, true)
```

Comment:
we will want a constant for the `true`. #5651 puts this in a common location for this package, so once that is merged we you will be able to use `persistent` and `!persistent` 😄